public/components/user/view-model.js

```diff
@@ -47,4 +47,11 @@ module.exports = can.Map.extend({
 			}
 		});
 	}
+	//, getList: function () {
+	//	return User.getList();
+	//},
+	//init: function () {
+	//	window.userVM = this;
+	//	//userVM.getList();
+	//}
```

Comment:
I deleted it then put it back - wasn't sure if it would be useful later. Will just delete things like this in the future for this project. Will push with it removed in a sec.